app/models/flaky_result_report.rb

```diff
@@ -0,0 +1,37 @@
+class FlakyResultReport
+
+  def self.populate( args )
+                    
+    primary_world_id = args[:world_id] or raise "Need to provide a primary world id"
+    primary_world = World.find(primary_world_id)
+    target = args[:target]
+    
+    tags = args[:tags]
+    
+    primary_results_by_test = Result.unscoped
+                                    .includes(:test_definition => :tags,
+                                      :children => [
```

Comment:
the formatting isn't consistent here. I don't know idiomatic Ruby, but I think it should be:

```
Result.unscoped
      .includes(
        :test_definition => :tags,
        :children => [
          :children => [:test_definition => :tags],
          :test_definition => :tags 
      ])
```

but it's not a huge deal.

```
```

Reply:
probably need to add a linter here so that we can detect these issues before commit

Reply:
I am ignoring this one for the linter to detect, we can add that as a feature separately.